tests/util.test.ts

```diff
@@ -14,54 +14,42 @@
  * limitations under the License.
  */
 
-import * as chai from 'chai';
-import chaiAsPromised from 'chai-as-promised';
+import { expect } from 'chai';
 import 'mocha';
-import * as sinon from 'sinon';
```

Comment:
I dropped sinon and interact directly with the filesystem. It's a better test and actually somehow faster in my benchmarks. Everything happens in tmpdir, so it's cleaned automatically by the OS too